Hangman/hangman.py

```diff
@@ -0,0 +1,55 @@
+from Generate_word import GenerateWord
+
+
+class Hangman:
+    def __init__(self, word="default"):
+        self.chance = 6
```

Comment:
Magic numbers

Reply:
Ok i will use store 6 in const and use that everywhere it is needed